apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/MerchantMonitoringTable.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+import { TBusinessReports } from '@/domains/business-reports/fetchers';
+import { DataTable, IDataTableProps } from '@/common/components/organisms/DataTable/DataTable';
+import { columns } from '@/pages/MerchantMonitoring/components/MerchantMonitoringTable/columns';
+
+export const MerchantMonitoringTable: FunctionComponent<{
+  data: TBusinessReports;
+}> = ({ data }) => {
+  const Cell: IDataTableProps<typeof data>['CellContentWrapper'] = ({ cell, children }) => {
+    return (
+      <span
+        // to={`/${locale}/profiles/individuals/${itemId}${search}`}
```

Comment:
Why it is commented?

Reply:
Convenience. Part of the task is creating the view the URL is leading to, once that in it will be uncommented and the span will be replaced by `Link`.